CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ FEATURES:
 BUG FIXES:
 
 * `azurerm_signar_service` - fixing a bug when expanding the `cors` block [GH-9671]
+* `azurerm_storage_account` - fixing an issue where `allow_blob_public_access` and `min_tls_version` are now available is US Government Cloud [GH-9128]
```

Comment:
We update the changelog by hand after merge so could we remove this?

Reply:
Removed, thanks.